version.txt

```diff
@@ -1 +1 @@
-v3.8.1
+v3.8.2
```

Comment:
I know technically we are not changing anything in the behavior of che actions but since we are doing quite a big change across all the actions we could actually ship it as v4.0.0